packages/logs/src/boot/startLogs.ts

```diff
@@ -88,6 +91,29 @@ export function doStartLogs(
   }
   errorObservable.subscribe(reportError)
 
+  if (isExperimentalFeatureEnabled('forward-reports')) {
+    trackReports(['deprecation', 'intervention'], (message: string, report: Report) => {
+      const loggerMethod = report.type === 'deprecation' ? 'warn' : 'error'
+      errorLogger[loggerMethod](message, { report: JSON.parse(JSON.stringify(report)) })
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏what is the reasoning for doing the stringify + parse?

Reply:
I use it to have a raw json object. With the web API `Report` object, our `jsonStringify` function used in the log batch return an empty object. I could have mapped the attributes one by one but for the naive implem I though it was easier. Do you have another idea?

Reply:
Interesting, no idea yet about the actual implementation however it could be interesting to reproduce this behavior with the stub.

**💬 suggestion:** ‏On a design perspective, it could be nice to also perform the report transformation inside the tracking function to avoid each trackReports user to do it (same with trackCsp).

Reply:
For the design suggestion, sure. 
Less sure for the `stringify` though. It's not a behaviour of the reports. It is a due to our `jsonStringify` implementation which removes the `toJSON` of the Report prototype.

Reply:
If this issue comes from our `jsonStringify` implementation, should not we fix it?
`JSON.parse(JSON.stringify(...))` seems really hacky to me.

Reply:
Reports provide a toJSON method. So I now use it.